### ISSUE TITLE
A few things to add to the documentation after the November users committee

### ISSUE THREAD

**Issue by lesteve** [jean-zay-users/jean-zay-doc#49]
@remicres @mdiazmel feel free to edit and add more if I missed some:
- [ ] ability to ask for CPU hours (it should stay a small fraction of GPU hours) in Dynamic Access. I guess we should try to check that we can find it in the eDARI form and do a screenshot or something
- [ ] compil machines. My understanding is that those are interactive machines where some of the limitations of r the login node does not apply. They were created mostly to do multi-thread compilation because on the login nodes you are restricted to use only one CPU via cgroups. Hopefully we'll have more details about these machines in one of the next Flash Info

**Comment by RemiLacroix-IDRIS** [comment]
> compil machines. My understanding is that those are interactive machines where some of the limitations of the login node does not apply. They were created mostly to do multi-thread compilation because on the login nodes you are restricted to use only one CPU via cgroups.

That's correct, except that they are not "interactive" as in you can SSH on them. You need to request some resources on it first via Slurm. The easiest way is to use something like `srun -c 5 -p compil --pty bash`.

We have added some info on our documentation (for example in http://www.idris.fr/eng/jean-zay/cpu/jean-zay-cpu-exec_partition_slurm-eng.html) and it will be in the next Flash-Info.

**Comment by lesteve** [comment]
OK thanks for the details! It is really great that you don't need CPU hours to use them.

On top of this, these machines have access to the internet. This is great too because I have heard of workflows (e.g. PlantNet) where they need to refresh the dataset from their external databases and they need both CPUs and interenet access.
